src/component/utility/FileUtility.ts

```diff
@@ -13,9 +13,11 @@ export const FILES_SIGNATURES = {
 };
 
 function getFileSignature(fileArrayBuffer) {
-  return new Uint8Array(fileArrayBuffer)
+  let uint = '';
+  new Uint8Array(fileArrayBuffer)
     .slice(0, 4)
-    .reduce((acc, byte) => (acc += byte.toString(16).padStart(2, '0')), '');
+    .forEach((byte) => (uint += byte.toString(16).padStart(2, '0')));
```

Comment:
`.map((byte) => ...).join('')`

Reply:
there's a rule that force Uint8Array.map to return number, should i ignore it?

Reply:
You can use `Array.from(uint8array, (byte) => '').join('')`

Reply:
thank you